roles/accounting/tasks/main.yml

```diff
@@ -0,0 +1,230 @@
+---
+# tasks file for accounting
+
+
+- name: Ensure group "{{ accounting_group }}" exists
+  group:
+    name: "{{ accounting_group }}"
+    state: present
+  tags:
+    - accounting
+    - accounting:user
+
+- name: Add user "{{ accounting_user }}"
+  user:
+    name: "{{ accounting_user }}"
+    groups: "{{ accounting_group }}"
+    append: yes
+    create_home: no
+    shell: /sbin/nologin
+    comment: accounting nologin user
+  tags:
+    - accounting
+    - accounting:user
+
+
+- name: Install Java 11
+  yum:
+    name: "{{ packages }}"
+    state: latest
+  vars:
+    packages:
+    - java-11-openjdk
+    - java-11-openjdk-devel
+  tags:
+    - accounting
+    - accounting:packages
+    - accounting:java
+    - accounting:maven
+
+- name: Download maven 3.8.4 and unarchive in /opt/
+  unarchive:
```

Comment:
This had issues throwing ssl_error with get_url python module and did need to add validate_certs: no. Even though the remote_src is set to `yes` its seems as the unarchive tries to download the file locally (can you verify this?)